base/process.jl

```diff
@@ -81,7 +88,15 @@ end
 
 immutable DevNullStream <: AsyncStream end
 const DevNull = DevNullStream()
+isreadable(::DevNullStream) = false
+iswritable(::DevNullStream) = true
+isopen(::DevNullStream) = true
+read{T<:DevNullStream}(::T, args...) = throw(EOFErorr())
+write{T<:DevNullStream}(::T, args...) = 0
```

Comment:
i think you're right, i just don't know if there is a good way to generically compute the number of bytes that got "written"